src/main/java/net/sacredlabyrinth/phaed/simpleclans/hooks/discord/exceptions/InvalidChannelException.java

```diff
@@ -0,0 +1,19 @@
+package net.sacredlabyrinth.phaed.simpleclans.hooks.discord.exceptions;
+
+import net.sacredlabyrinth.phaed.simpleclans.SimpleClans;
+import org.jetbrains.annotations.NotNull;
+
+import static net.sacredlabyrinth.phaed.simpleclans.managers.SettingsManager.ConfigField.DEBUG;
+
+public class InvalidChannelException extends Exception {
```

Comment:
extends DiscordHookException

Reply:
If I'll extend it, it would break the logic [here](https://github.com/RoinujNosde/SimpleClans/pull/281/files#diff-59dd7bda205c34ef18d1a41d5558eadb390d259e9ad3ea256dfc8e58ad6881ecR582)

Reply:
How?